tests/unittests/AddressBook/BasicQueryTests.mm

```diff
@@ -0,0 +1,115 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+
+#import <CoreFoundation/CoreFoundation.h>
+#import <Foundation/Foundation.h>
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABAddressBook.h>
+#import <AddressBook/ABRecord.h>
+#import <AddressBook/ABPerson.h>
+#import "ABContactInternal.h"
+
+class AddressBookQueryTest : public ::testing::TestWithParam<::testing::tuple<ABPropertyID, NSString*>> {
+protected:
+    virtual void SetUp() {
+        WACContact* contact = [WACContact make];
+        contact.notes = @"Test Note!";
+        contact.nickname = @"Mikey";
+        contact.yomiGivenName = @"Mic";
+        contact.honorificNameSuffix = @"Jr.";
+        contact.yomiFamilyName = @"Saw-ft";
+        contact.middleName = @"Roe";
+        contact.lastName = @"Soft";
+        contact.honorificNamePrefix = @"Mr.";
+        contact.firstName = @"Mike";
+
+        WACContactJobInfo* job = [WACContactJobInfo make];
```

Comment:
>make [](start = 52, length = 4)

ARC doesn't appear to be on for the test files. I think you are probably leaking these things?